app/models/agents/gap_detector_agent.rb

```diff
@@ -8,6 +8,11 @@ class GapDetectorAgent < Agent
       The `value_path` value is a [JSONPath](http://goessner.net/articles/JsonPath/) to a value of interest. If either
       this value is empty, or no Events are received, during `window_duration_in_days`, an Event will be created with
       a payload of `message`.
+
+      The `stream_path` value is a [JSONPath](http://goessner.net/articles/JsonPath/) to a value in the event that will
```

Comment:
I have to turn the question around and ask why it does make sense.  Your contention seems to be, "because there are two parameters whose names end in _path."  But these are used for entirely different concepts which could be used together.  Neither of us have come up with use case for doing so, but it doesn't mean someone won't.  Would you ever do this in code?  Would you make a method perform two distinctly different operations by passing in a modal flag?